packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/Page.tsx

```diff
@@ -188,25 +191,16 @@ export const Page = ({ editingContextId, representationId, formDescriptionEditor
     <div className={classes.preview}>
       <div className={classes.page}>
         {page.groups.map((group) => {
-          return (
-            <Group
-              key={group.id}
-              editingContextId={editingContextId}
-              representationId={representationId}
-              formDescriptionEditor={formDescriptionEditor}
-              page={page}
-              group={group}
-            />
-          );
+          return <Group key={group.id} page={page} group={group} />;
         })}
       </div>
       <div
         data-testid="Page-DropArea"
         className={classes.bottomDropArea}
-        onDragEnter={handleDragEnter}
-        onDragOver={handleDragOver}
-        onDragLeave={handleDragLeave}
-        onDrop={handleDrop}>
+        onDragEnter={readOnly ? noop : handleDragEnter}
```

Comment:
May be it would have been better/more readable to modify handleDragEnter/handleDragOver/handleDragLeave... instead.
What do you think about it?